nodemon.json

```diff
@@ -1,5 +1,11 @@
 {
   "watch": [".public", "./src"],
-  "ignore": ["**/*.test.*", "./test-helpers", "./src/client"],
+  "ignore": [
+    "**/*.test.*",
+    "./test-helpers",
+    "./src/client",
+    "src/server/common/components",
+    "src/server/common/templates/partials"
```

Comment:
Feels a bit odd having both ways writing the paths. I'd say pick 1 and use it all over:

```
"./**/*.test.*",
"./test-helpers",
"./src/client",
"./src/server/common/components",
"./src/server/common/templates/partials"
```

Or


```
"**/*.test.*",
"test-helpers",
"src/client",
"src/server/common/components",
"src/server/common/templates/partials"
```

Reply:
And to double check - are changes in clientSide work in either `src/client` or SASS or JS found within `src/server/common/components` or  `src/server/common/templates/partials` picked up by webpack as expected?